src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.MetadataUpdate.cs

```diff
@@ -145,23 +147,24 @@ static int[] ReadIntArray(BinaryReader binaryReader)
 
 		private static void ReloadWithUpdatedTypes(Type[] updatedTypes)
 		{
-			foreach (var updatedType in updatedTypes)
+			if (updatedTypes.Length == 0)
 			{
-				if (_log.IsEnabled(LogLevel.Debug))
-				{
-					_log.LogDebug($"Processing changed type [{updatedType}]");
```

Comment:
Did we lose that logging? Could it be plugged in back in the new logic? Maybe at the start of `ReplaceViewInstance`?

Reply:
We lost it because we're not processing each updateType in order any more. Now we're iterating the visual tree and just updating any element that needs updating